db.json

```diff
@@ -0,0 +1,48 @@
+{
+  "posts": [
+    {
+      "html": "<p><strong>Hello world!!!</strong></p>",
+      "id": 1
+    },
+    {
+      "html": "<p><strong>Hello world!!!</strong>fgsfg</p>",
+      "id": 2
+    },
+    {
+      "html": "<p>dfdfaee</p>",
+      "id": 3
+    },
+    {
+      "html": "<p><strong>Hello world!!!</strong></p>\n<p>where is my </p>\n<p>line break???</p>",
+      "id": 4
+    },
+    {
+      "html": "<p><strong>Hello world!!!</strong></p>\n<p>no line breakk</p>\n<p>for heaven sake!</p>\n<pre><code>this is \n   code, man\n</code></pre>",
+      "id": 5
+    },
+    {
+      "html": "<p><strong>Hello world!!!</strong></p>\n<h1 id=\"headerhere\">header here</h1>\n<p><strong>bold</strong></p>\n<p><em>italic</em></p>\n<p><del> line-through</del></p>\n<p><code>single line of code</code> normal text</p>",
+      "id": 6
+    },
+    {
+      "html": "<p>some code picture</p>\n<p><img src=\"https://images.unsplash.com/photo-1515879218367-8466d910aaa4?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80\" alt=\"\" /></p>",
+      "id": 7
+    },
+    {
+      "html": "<p><strong>Hello world!!!</strong></p>\n<pre><code class=\"java language-java\">java code\n</code></pre>",
+      "id": 8
+    },
+    {
+      "html": "<p>**hello ***</p>",
+      "id": 9
+    },
+    {
+      "html": "<h1 id=\"thisisheader\">this is header :)</h1>\n<p>this is normal paragraph with <code>some code in it</code></p>\n<p>there should be a empty line above</p>\n<pre><code>// this is code snippet\nfunction beGood(){\n  return 'not bad'\n}\n</code></pre>",
+      "id": 10
+    },
+    {
+      "html": "<p><strong>still works</strong></p>",
+      "id": 11
+    }
+  ]
+}
```

Comment:
Missing EOF at the end of file. Please configure your IDE to add this automatically

https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files

Reply:
done